content/en/cloud_cost_management/cost_allocation/bigquery.md

```diff
@@ -0,0 +1,112 @@
+---
+title: BigQuery Cost Allocation
+description: Learn how to allocate Cloud Cost Management spending across your organization with BigQuery Cost Allocation.
+further_reading:
+  - link: "/cloud_cost_management/"
+    tag: "Documentation"
+    text: "Learn about Cloud Cost Management"
+---
+
+## Overview
+
+Datadog Cloud Cost Management (CCM) automatically allocates the costs of your Google BigQuery resources to individual queries and workloads. Use cost metrics enriched with tags from queries, projects, and reservations to visualize BigQuery workload costs in the context of your entire cloud bill.
+
+## Resources
```

Comment:
I am not sure what this section is adding? Maybe it can be combined with the overview if there is anything important, otherwise I feel like the rest of the doc covers this. 
just my opinion, feel free to ignore me